site/content/extend/customization/_index.md

```diff
@@ -0,0 +1,24 @@
+---
+title: "Customizing Mattermost"
+date: "2017-10-29T13:47:55+09:00"
+section: "extend"
+---
+
+Mattermost allows for a variety of customization options and variations, making it possible to create a more adequate experience depending on the needs of each deployment.
```

Comment:
"Mattermost allows for a variety of customization options and **modifications**" ...